Docs for Deleting Repo Host STS Following Upgrade from 5.0.0-5.0.2 (#2859)
CrunchyData/postgres-operator: docs/content/installation/upgrade.md

```bash
kubectl apply -k kustomize/install/bases
```

> ## Upgrading from PGO 5.0.0-5.0.2

Reviewer: ```suggestion
## Upgrading from PGO v5.0.2 and below
```

> As a result of changes to pgBackRest dedicated repository host deployments in PGO 5.0.3

Reviewer: ```suggestion
As a result of changes to pgBackRest dedicated repository host deployments in PGO v5.0.3
```

> (please see the [PGO 5.0.3 release notes]({{< relref "../releases/5.0.3.md" >}}) for more details),

Reviewer: ```suggestion
(please see the [PGO v5.0.3 release notes]({{< relref "../releases/5.0.3.md" >}}) for more details),
```

reconciliation of a pgBackRest dedicated repository host might become stuck with the following

> error (as shown in the PGO logs) following an upgrade from PGO versions 5.0.0-5.0.2:

Reviewer: ```suggestion
error (as shown in the PGO logs) following an upgrade from PGO versions v5.0.0 through v5.0.2:
```

```bash
StatefulSet.apps \"hippo-repo-host\" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', 'updateStrategy' and 'minReadySeconds' are forbidden
```

If this is the case, proceed with deleting the pgBackRest dedicated repository host StatefulSet,
and PGO will then proceed with recreating and reconciling the dedicated repository host normally:

```bash
kubectl delete sts hippo-repo-host
```

Additionally, please be sure to update and apply all PostgresCluster custom resources in accordance
with any applicable spec changes described in the 

> [PGO 5.0.3 release notes]({{< relref "../releases/5.0.3.md" >}}).

Reviewer: ```suggestion
[PGO v5.0.3 release notes]({{< relref "../releases/5.0.3.md" >}}).
```
